packages/faucet/src/api/webserver.ts

```diff
@@ -14,8 +14,14 @@ export interface ChainConstants {
   readonly chainId: string;
 }
 
+export interface AddressEntry {
+  address: string;
+  date: number;
```

Comment:
I think this makes the code a bit more intuitive to read

```suggestion
  lastUsed: Date;
```